.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+
+name: Python package
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox wheel
+        pip install .
+    - name: Test  
+      run: tox
+    - name: Build
+      run: |
+        python setup.py sdist bdist_wheel
+        python -m pip install --find-links=dist --no-index --ignore-installed --no-deps importnb
+    - name: Publish package
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
can probably hoist this up to on...

Reply:
can i skip this logic if i use the published trigger?

Reply:
yeah, that's the idea!